docs/samples/serial.md

```diff
@@ -0,0 +1,83 @@
+## Using the Twitch sample bundle
```

Comment:
Twitch copy-pastes are a meme at this point :laughing: 

```suggestion
## Using the serial sample bundle
```

Reply:
Thanks, should have grepped before opening the pr